app/views/sign_up/recovery_codes/_recovery_page_controller.js.erb

```diff
@@ -0,0 +1,96 @@
+var modalSelector = '#personal-key-confirm';
+var modal = new LoginGov.Modal({el: modalSelector });
+
+var recoveryWords = [].slice.call(document.querySelectorAll('[data-recovery]'));
+var formEl = document.getElementById('confirm-key');
+var inputs = [].slice.call(formEl.elements).filter(function(el) { return el.type === 'text'});
+var modalTrigger = document.querySelector('[data-toggle="modal"]');
+var modalDismiss = document.querySelector('[data-dismiss="personal-key-confirm"]')
+
+var reminderEl = document.getElementById('recovery-code-reminder');
+
+var isInvalidForm = false;
+var alertFallback = '<div class="alert alert-alert" role="alert" id="recovery-code-alert">' +
+                    "<%= t('users.recovery_code.confirmation_error') %>" +
+                    '</div>';
+
+function show(event) {
+  event.preventDefault();
+
+  modal.on('show', function() {
+    inputs[0].focus();
+  });
+
+  modal.show();
+}
+
+function hide() {
+  modal.on('hide', function() {
+    resetForm();
+
+    if (reminderEl.className.indexOf('hide')) {
+      reminderEl.setAttribute('aria-hidden', false);
+      reminderEl.className = reminderEl.className.replace(/\s+hide/, '');
```

Comment:
I added a classlist polyfill already, maybe we could just toggle the `hide` class instead?